static/src/stylesheets/module/facia/_cp-scott.scss

```diff
@@ -0,0 +1,75 @@
+.fc-cp-scott__quote {
+    display: block;
+    color: colour(neutral-2);
+}
+
+.fc-cp-scott__quote-line {
+    display: inline-block;
+
+    @include mq(mobileLandscape) {
+        display: block;
+    }
+}
+
+.fc-cp-scott__citation {
+    display: block;
+}
+
+.fc-cp-scott__portrait {
+    float: right;
+    width: 70px;
+
+    @include mq(tablet) {
+        margin-top: -$gs-baseline / 2;
+        margin-bottom: $gs-baseline / 2;
+        width: 80px;
+    }
+
+    @include mq(leftCol) {
+        margin-top: 0;
+        margin-bottom: 0;
+        float: none;
+    }
+
+    @include mq(wide) {
+        .has-page-skin & {
+            float: right;
+        }
+    }
+}
+
+.fc-cp-scott__text {
+    clear: left;
+    float: left;
+
+    @include fs-headline(1);
+    line-height: $gs-baseline * 4 / 3;
+
+    .inline-quote svg {
+        height: $gs-gutter / 2;
+        fill: colour(neutral-2);
+    }
+
+    @include mq(mobileLandscape) {
+        clear: none;
+        float: right;
+        margin-right: $gs-baseline;
+        margin-top: 2px;
+    }
+
+    @include mq(tablet) {
+        margin-top: 0;
+    }
+
+    @include mq(leftCol) {
+        float: none;
+        margin-right: 0;
+    }
+
+    @include mq(wide) {
+        .has-page-skin & {
+            float: right;
+            margin-right: $gs-baseline;
```

Comment:
Can't use `gs-baseline` for left or right. Use `gs-gutter / 2` to get similar output but still on grid.